src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java

```diff
@@ -30,7 +33,7 @@ public void should_find_no_Property_if_repository_is_empty(){
     @Test
     public void should_store_a_Property() {
         //Creating
-        PropertyEntity propertyToSave = UtilsTest.createPropertyEntityWithBasicFields();
+        PropertyEntity propertyToSave = EntityUtilsTest.createPropertyEntityWithAllFields();
```

Comment:
we are testing with all fields in the test should_store_a_Property_With_All_Nested_Fields, here we can test to save a property just with basic fields